option optimize_for = LITE_RUNTIME;

message LegionBossTaskGetRequest
{
 required uint32 id = 1;
}
